frontend/ui/widget/dictquicklookup.lua

```diff
@@ -1158,14 +1158,7 @@ function DictQuickLookup:changeDictionary(index, skip_update)
         self.displaynb = T("%1 / %2", index, #self.results)
         -- add queried word to 1st result's definition, so we can see
         -- what was the selected text and if we selected wrong
-        if index == 1 then
-            if self.is_html then
-                self.definition = self.definition.."<br/>_______<br/>"
-            else
-                self.definition = self.definition.."\n_______\n"
-            end
-            self.definition = self.definition..T(_("(query : %1)"), self.word)
-        end
+        self:addQueryWordToFirstResult(index)
```

Comment:
Feels a bit odd, to have that method named addQueryWordToFirstResult(), pass it the index so it can check that only when index==1, it can addQueryWord.
I would keep the `if index == 1 then` here and call `self:addQueryWordToResult()`.